src/rust/lib/system/web/src/lib.rs

```diff
@@ -497,74 +495,6 @@ extern "C" {
 // === Printer ===
 // ===============
 
-type PrintFn = fn(&str) -> std::io::Result<()>;
-
-struct Printer {
-    printfn: PrintFn,
-    buffer: String,
-    is_buffered: bool,
-}
-
-impl Printer {
-    fn new(printfn: PrintFn, is_buffered: bool) -> Printer {
-        Printer {
-            buffer: String::new(),
-            printfn,
-            is_buffered,
-        }
-    }
-}
-
-impl std::io::Write for Printer {
-    fn write(&mut self, buf: &[u8]) -> std::io::Result<usize> {
-        self.buffer.push_str(&String::from_utf8_lossy(buf));
-
-        if !self.is_buffered {
-            (self.printfn)(&self.buffer)?;
-            self.buffer.clear();
-
-            return Ok(buf.len());
-        }
```

Comment:
Please provide `println!` mcro that works in the web (that uses `console.log` user the hood) in this PR, as part of our prelude. Otherwise, this will make development a lot harder and will cause trouble among all devs.

Reply:
Non trivial, as we have no proper means of hiding the default `println!`, adding a new one to prelude just makes them ambiguous.
Also, this cannot be part of this PR, since prelude belongs to a separate repo.
To be discussed how to handle this.